tests/harness/executor/executor.go

```diff
@@ -53,3 +54,7 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func TestEverything(test *testing.T) {
```

Comment:
Is this some Go test thing that gets called automatically?

Reply:
Yes, `go_test` bazel rules use the [Go test package](https://golang.org/pkg/testing/), which runs everything that looks like `TestXXX(arg *testing.T)`.